examples/udp_client/linux_udp_client.cc

```diff
@@ -0,0 +1,135 @@
+/**************************************************************************/ /**
+ * @brief Simple Linux UDP client example.
+ * @file
+ *
+ * @note
+ * This is a minimal UDP client implementation, meant as an example of how to
+ * connect to a device and decode incoming data. It is not robust to network
+ * outages, socket reconnects, or other typical network errors. Production
+ * implementations must check for and handle these cases appropriately (by
+ * checking expected `errno` values, etc.).
+ ******************************************************************************/
+
+#include <cerrno>
+#include <cmath> // For lround()
+#include <csignal> // For signal()
+#include <cstdio> // For fprintf()
+#include <cstring> // For memcpy()
+#include <string> // For stoi() and strerror()
+
+#include <netdb.h> // For gethostbyname() and hostent
+#include <netinet/in.h> // For IPPROTO_* macros and htons()
+#include <sys/socket.h> // For socket support.
+#include <unistd.h> // For close()
+
+#include <iostream>
+#include <fstream>
+#include <arpa/inet.h>
+
+#include <point_one/fusion_engine/parsers/fusion_engine_framer.h>
+
+#include "../common/print_message.h"
+
+#define DEBUG_ON 0
+
+using namespace point_one::fusion_engine::examples;
+using namespace point_one::fusion_engine::parsers;
+
+static bool shutdown_pending_ = false;
+
+/******************************************************************************/
+void HandleSignal(int signal) {
+  if (signal == SIGINT || signal == SIGTERM) {
+    std::signal(signal, SIG_DFL);
+    shutdown_pending_ = true;
+  }
+}
+
+void * get_in_addr(struct sockaddr * sa) {
+    if (sa->sa_family == AF_INET) {
+        return &(((struct sockaddr_in *)sa)->sin_addr);
+    }
+    return &(((struct sockaddr_in6 *)sa)->sin6_addr);
+}
+/******************************************************************************/
+
+int main(int argc, const char* argv[]) {
+  // Parse arguments.
+  if (argc > 2) {
+    printf(R"EOF(
+Usage: %s [PORT]
+
+Connect to an Atlas device over UDP and print out the incoming message
+contents.
+)EOF", 
+        argv[0]);
+    return 0;
+  }
+
+  int port = (argc == 2) ? std::stoi(argv[1]) : 12345;
+
+  // create UDP socket.
+  int sock = socket(AF_INET, SOCK_DGRAM, IPPROTO_UDP);
+  if (sock < 0) {
+    printf("Error creating socket.\n");
+    return 2;
+  }
+
+  // bind socket to port
```

Comment:
Same here: capital "Bind" and add a period. We like to make everything complete sentences wherever possible for consistency and readability.

Reply:
done.